src/Orleans/Logging/LoggerWrapper.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+
+namespace Orleans.Runtime
+{
+    //TODO: Mark it as [Obsolete] and internal after all runtime has migrated
+    public class LoggerWrapper<T> : Logger
```

Comment:
should this be `LegacyLogger<T>` ?

Reply:
why? application will not be interacting with this directly . they will be interacting with Logger. 
What's your concern about its name?

Reply:
expecting users to use a Wrapper (when it's not entirely clear wrapper for what) feels bad. On the other hand, this is the Logger wrapper we keep for legacy reasons, so that's why I suggested that name.
Alternatively I propose that this is renamed to `Logger<T>`, and also that we follow suit and have an extension method `loggingFactory.CreateLegacyLogger<T>()` that returns a Logger. The user never has to know the implementation details of how we wrap things